routers/api/v1/user/repo.go

```diff
@@ -66,6 +70,24 @@ func ListMyRepos(ctx *context.APIContext) {
 	listUserRepos(ctx, ctx.User)
 }
 
+// ListOrgRepos - list the repositories of an organization.
+func ListOrgRepos(ctx *context.APIContext) {
+	// swagger:route GET /orgs/{org}/repos orgListRepos
+	//
+	//     Produces:
+	//     - application/json
+	//
+	//     Responses:
+	//       200: RepositoryList
+	//       500: error
+
+	org := GetOrgByParams(ctx)
```

Comment:
Not necessary, can just use `ctx.Org` (which is populated by the `orgAssignment(true)` handler)